src/osd/OSD.cc

```diff
@@ -2512,6 +2513,25 @@ void OSD::create_logger()
 
   PerfCountersBuilder osd_plb(cct, "osd", l_osd_first, l_osd_last);
 
+  // Latency axis configuration for op histograms, values are in nanoseconds
+  PerfHistogramCommon::axis_config_d op_hist_x_axis_config{
+    "Latency (usec)",
+    PerfHistogramCommon::SCALE_LOG2, ///< Latency in logarithmic scale
+    0,                               ///< Start at 0
+    100000,                          ///< Quantization unit is 100usec
```

Comment:
Looking at HdrHistogram it looks like the space requirements will be significantly higher if you want that level of precision, right?

I haven't looked at this is much detail, but I'm confused what the quantization value is used for... isn't the data effectively quantized by the bin size anyway, which is logarithmic?

If we were to swap in HdrHistogram (or similar) later, would/can the reporting interface remain the same?